subcommands/waves/init.go

```diff
@@ -27,7 +27,12 @@ func init() {
 This command only initializes a wave, but does not provision its updates to devices.
 Use a "fioctl wave rollout <wave> <group>" to trigger updates of this wave to a device group.
 Use a "fioctl wave complete <wave>" to update all devices (make it globally available).
-Use a "fioctl wave cancel <wave> to cancel a wave (make it no longer available).`,
+Use a "fioctl wave cancel <wave> to cancel a wave (make it no longer available).
+
+We recommend that you generate static deltas for your production targets to optimize 
+OTA update download. Consider generating a static delta for targets using:
```

Comment:
We use the [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/) in help messages.
Please, see the related section in the [style guide](https://foundriesio.atlassian.net/wiki/spaces/ID/pages/2392067/Foundries.io+Style+and+Communication+Guide#General-Guidelines).

```suggestion
We recommend that you generate static deltas for your production targets to optimize OTA update download.
Consider generating a static delta for targets using:
```

The line length is absolutely irrelevant here, as long as the sentence fits within [25 words](https://foundriesio.atlassian.net/wiki/spaces/ID/pages/2392067/Foundries.io+Style+and+Communication+Guide#Sentence-Length).